server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -269,17 +267,25 @@ export function configureWebSocketServices(
 		async function connectDocument(message: IConnect): Promise<IConnectedClient> {
 			const startTime = Date.now();
 			const throttleErrorPerCluster = checkThrottleAndUsage(
-				connectThrottlerPerCluster,
+				throttlersMap?.get("generalCluster")?.get("socketConnections"),
```

Comment:
just confirming - Does checkThrottleAndUsage() gracefully handle null or undefined throttler?
It might be a good idea to test with empty throttler or missing throttling in map or empty throttlerMap in config ?

Reply:
Yes, the checkThrottleAndUsage() will directly return for handling null or undefined throttler. 
Current throttlerMap is hard-coded, so it is less chance to let it be null or empty.